src/pro/plugins/stripe-checkout/index.js

```diff
@@ -0,0 +1,59 @@
+import { Fragment } from '@wordpress/element';
```

Comment:
Same here, we can segregate dependencies for better readability. It's a NIT and not blocking.